src/sentry/api/endpoints/organization_details.py

```diff
@@ -997,7 +998,20 @@ def put(self, request: Request, organization: Organization) -> Response:
                 if is_org_mode and (
                     "samplingMode" in changed_data or "targetSampleRate" in changed_data
                 ):
-                    boost_low_volume_projects_of_org_with_query.delay(organization.id)
+                    # use 1 day as the default interval for project sampling rate rebalancing
```

Comment:
This code branch handles the switch to automatic mode and when the target rate is changed. See the comment above the branch. 

You'll want to modify the logic inside `_compute_project_target_sample_rates` instead.